BingMaps/rest-services/examples/driving-route-with-route-path-example.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Driving Route with Route Path Example | Microsoft Docs"
+description: "This example returns a driving route from the Minneapolis, Minnesota to St. Paul, Minnesota. Results are shown for both JSON and XML response formats."
```

Comment:
This example returns a driving route from Minneapolis, Minnesota to St. Paul, Minnesota.